### PR TITLE
docs(aap): aap backend plugin documentation

### PR DESCRIPTION
**Pull request by hmanwani-rh**
GH issue: https://github.com/janus-idp/backstage-plugins/issues/546

### CODE REVIEW

**Review by tumido — COMMENTED** [review]
Hi @hmanwani-rh , 

can you please run `yarn prettier:fix`? The `README.md` doesn't comply with our linting rules.

**Comment by hmanwani-rh** [comment]
> Hi @hmanwani-rh ,
> 
> can you please run `yarn prettier:fix`? The `README.md` doesn't comply with our linting rules.

Done, but still the build is failing.

File: plugins/aap-backend/README.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
-# AAP Backstage provider
+# Ansible Automation Platform Backstage provider plugin
 
-The AAP Backstage provider plugin synchronizes the AAP job templates and workflow job templates into the [Backstage](https://backstage.io/) catalog.
+The Ansible Automation Platform (AAP) Backstage provider plugin synchronizes the AAP content including job templates into the [Backstage](https://backstage.io/) catalog.
```
Comment (invincibleJai) [inline plugins/aap-backend/README.md:3]: we show job templates and workflow job templates in catalog if we want to mention this explictly 

this was added with https://github.com/janus-idp/backstage-plugins/pull/674

File: plugins/aap-backend/README.md [hunk 2]
```diff
@@ -104,3 +104,36 @@ When you start your Backstage application, you can see some log lines as follows
...
+
+#### Prerequisites
+
+- Your Backstage application is installed and running.
+- You have installed the AAP Backstage provider plugin. For installation and configuration instructions, see [Installation and configuration](#installation-and-configuration).
+
+#### Procedure
+
+1. Open your Backstage application and Go to the **Catalog** page.
+1. Select **Resource** from the **Kind** drop-down and **job_template** from the **Type** drop-down on the left side of the page.
```
Comment (invincibleJai) [inline plugins/aap-backend/README.md:122]:
```suggestion
1. Select **Resource** from the **Kind** drop-down and **job template** or **workflow job template** from the **Type** drop-down on the left side of the page.
```

File: plugins/aap-backend/README.md (hunk 1 above)
Comment (hmanwani-rh) [reply plugins/aap-backend/README.md]: Done

File: plugins/aap-backend/README.md (hunk 2 above)
Comment (hmanwani-rh) [reply plugins/aap-backend/README.md]: Done

**Review by invincibleJai — APPROVED** [review]
LGTM

**Comment by invincibleJai** [comment]
@tumido do you have insights into why are build failing here https://github.com/janus-idp/backstage-plugins/actions/runs/6048851995/job/16414995269?pr=690?

**Review by tumido — APPROVED** [review]

**Pull request merged**